bin/cron/update_hoc_map

```diff
@@ -50,6 +47,9 @@ def main
         data['organization_name_s']
       end
 
+    next if organization_name.nil? || processed_data['location_p'].nil?
```

Comment:
Note: the only change made in this PR other than moving these lines is changing `data['organization_name_s'].nil?` to `organization_name.nil?`